roles/common/tasks/ssh.yaml

```diff
@@ -21,5 +12,5 @@
   copy:
     src: files/ssh_config
     dest: /etc/ssh/ssh_config
-    mode: "0644"
+    mode: "0600"
```

Comment:
Leave mode "644" for client configuration. Otherwise, all users except root who want to ssh from an instance with this configuration to another one(s) will probably have difficulties. Therefore, please return everything as it was before.

Reply:
Done